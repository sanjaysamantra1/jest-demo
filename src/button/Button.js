import React from "react";

export default function Button() {
  return (
    <>
      <button onClick={()=>{console.log('hiiiiiiiiii')}}>Click me-1</button>
    </>
  );
}
