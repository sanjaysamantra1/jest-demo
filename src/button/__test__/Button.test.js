import renderer from "react-test-renderer";
import Button from "../Button";

it("renders Button Component correctly", () => {
  const tree = renderer.create(<Button />).toJSON(); // DOM Tree
  expect(tree).toMatchSnapshot();
});