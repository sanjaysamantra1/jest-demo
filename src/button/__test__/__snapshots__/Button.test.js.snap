// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Button Component correctly 1`] = `
<button
  onClick={[Function]}
>
  Click me-1
</button>
`;
